devsetup/scripts/tripleo.sh

```diff
@@ -35,16 +36,45 @@ if [[ ! -f $SSH_KEY_FILE ]]; then
     exit 1
 fi
 
+sudo dnf install -y python-jinja2
 source ${SCRIPTPATH}/common.sh
 
+# Clock synchronization is important for both Ceph and OpenStack services, so both ceph deploy and tripleo deploy commands will make use of chrony to ensure the clock is properly in sync.
+# We'll use the NTP_SERVER environmental variable to define the NTP server to use.
+# If we are running alls these commands in a system inside the Red Hat network we should use the clock.corp.redhat.com server:
+# export NTP_SERVER=clock.corp.redhat.com
+# And when running it from our own systems outside of the Red Hat network we can use any available server:
+# export NTP_SERVER=pool.ntp.org
+
+if [ $EDPM_COMPUTE_CELLS -eq 2 ] || [ $EDPM_COMPUTE_CELLS -gt 3 ] || [ $EDPM_COMPUTE_CELLS -eq 0 ] ; then
+    echo "Only a main cell0 plus a 2 additional compute cells supported yet!"
+    exit 1
+fi
+
+# TODO(bogdando): multi-cell with ceph HCI
+if [ $EDPM_COMPUTE_CELLS -eq 3 ] && [[ "$EDPM_COMPUTE_CEPH_ENABLED" =~ "rue" ]]; then
+    echo "Ceph storage is not supported for multi-cell setup yet! Deploying with a local storage."
+    COMPUTE_CEPH_ENABLED=False
+    EDPM_COMPUTE_CEPH_ENABLED=False
+    CEPH_ARGS=" "
+fi
+
 if [[ -e /run/systemd/resolve/resolv.conf ]]; then
     HOST_PRIMARY_RESOLV_CONF_ENTRY=$(cat /run/systemd/resolve/resolv.conf | grep ^nameserver | grep -v "${EDPM_COMPUTE_NETWORK_IP%.*}" | head -n1 | cut -d' ' -f2)
 else
     HOST_PRIMARY_RESOLV_CONF_ENTRY=${HOST_PRIMARY_RESOLV_CONF_ENTRY:-$GATEWAY}
 fi
 
+cat <<EOF > $MY_TMP_DIR/.standalone_env_file
+export RH_REGISTRY_USER="$REGISTRY_USER"
+export RH_REGISTRY_PWD="$RH_REGISTRY_PWD"
```

Comment:
if we run this in ci we may leak the user and pass for the registry...

Reply:
this is intented for local setups, but I will add ``set +x``

Reply:
even with +x i don't think this is acceptable since you are actually writing the user/pass to file here

this script is being executed by the existing multinode ci jobs i.e. not just for your new multi cell addition.

Isn't the whole point of this work that we will be adding a new multi cell job in ci? So what do you mean about local setups.